doc/api/cli.md

```diff
@@ -700,6 +700,20 @@ They are omitted from the values.
 USERNAME="nodejs" # will result in `nodejs` as the value.
 ```
 
+Multi-line values are supported:
```

Comment:
You need to include on top `changes: PR-URL: https://github.com/nodejs/node/pull/51289` and a description such as `add support to multi-line values`

Reply:
Could you check now pls ?